var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -134,6 +135,7 @@ class Boost(Package):
         "log",
         "math",
         "mpi",
+        "mqtt5",
```

Comment:
hash2 is also a new library. I'll leave it to you if you want to add it. I tend not to.

Reply:
Indeed. I left it out because it's header only (`--with-libraries=hash2` fails). Should it be added anyway? :thinking: